dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -47,9 +48,10 @@ public static void main(String[] args)
             System.exit(1);
         Class<?> runClass = loadCommand(allClasses, ns.get(Constants.COMMAND));
         DockerfileGitHubUtil dockerfileGitHubUtil = initializeDockerfileGithubUtil(ns.get(Constants.GIT_API));
+        DockerfileS3Util dockerfileS3Util = initializeDockerfileS3Util();
 
         /* Execute given command. */
-        ((ExecutableWithNamespace)runClass.newInstance()).execute(ns, dockerfileGitHubUtil);
+        ((ExecutableWithNamespace)runClass.newInstance()).execute(ns, dockerfileGitHubUtil, dockerfileS3Util);
```

Comment:
We can avoid this too if we detect the store type and create the right instance of our Store interface.